scripts/tick-cluster.js

```diff
@@ -527,10 +586,14 @@ function send(host, arg1, arg2, arg3, callback) {
 
 program
     .version(require('../package.json').version)
-    .arguments('<size>')
+    .option("-i, --interpreter [interpreter]")
```

Comment:
If <program> and <size> are required params, where do we validate them? How about providing a sane default for "size" and error out if no program is specified?